src/createProjector.ts

```diff
@@ -128,7 +141,19 @@ const createProjector: ProjectorFactory = createWidgetBase
 					};
 				});
 
-				projectorData.projector.append(projectorData.root, render);
+				switch (type) {
+					case 'append':
```

Comment:
do you think these should be symbols (something like `projector.attach({ type: projector.attachTypes.MERGE })`? or overkill?